style.css

```diff
@@ -0,0 +1,717 @@
+  flex-flow: row wrap;
+  margin-bottom: 26px;
+}
+
+.catalog__item {
+  width: 20%;
+  height: 261px;
+  min-width: 205px;
+}
+
+.catalog__item_1 {
+  margin-right: 3.5%;
+}
+
+.catalog__item_1:before{
+  position: relative;
+  content: "";
+  background: #000;
+  opacity: 0.2;
+  width: 1px;
+  height: 100.18px;
+  float: right;
+  top: 42px;
+  left: 15%;
+}
+
+.catalog__img {
+  margin-left: 5%;
+  margin-bottom: 23px;
+}
+
+.catalog__link {
+  display: block;
+  font-weight: normal;
+  line-height: 16px;
+}
+
+.catalog__wrap {
+  display: flex;
+  justify-content: space-between;
+}
+
+.catalog__price {
+  display: inline;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 24px;
+  width: 30%;
+}
+
+.catalog__scale {
+  display: inline-flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 25%;
+}
+
+.catalog button {
+  border: none;
+  padding: 0;
+  border-radius: 50%;
+}
+
+.catalog__round_grey {
+  background: #C4C4C4;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_black {
+  background: #0E1211;
+  width: 9px;
+  height: 9px;
+}
+
+.catalog__round_red {
+  background: #B62E3C;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_yellow {
+  background: #FDB431;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_peach {
+  background: #E29D69;
+  width: 9px;
+  height: 9px;
+}
+
+.catalog__round_wine {
+  background: #762E31;
+  width: 9px;
+  height: 9px;
+}
+
+.catalog__round_blue {
+  background: #41479B;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_azure {
+  background: #257DA9;
+  width: 9px;
+  height: 9px;
+}
+
+.video {
+  display: flex;
+  justify-content: space-around;
+  background: #000;
+  height: 494px;
+  color: #FFF;
+}
+
+.video__play {
+  width: 53.7%;
+  height: 345px;
+  background: url(images/video.png) no-repeat;
+  background-size: cover;
+  border: 8px solid rgba(255, 255, 255, 0.8);
+  text-align: center;
+}
+
+.video__play:before {
+  display: inline-block;
+  height: 100%;
+  vertical-align: middle;
+  content: "";
+}
+
+.video__btn {
+  background: url(images/play.png) no-repeat;
+  background-size: cover;
+  height: 46.8px;
+  width: 46.8px;
+  /*border: 5px solid rgba(255, 255, 255, 0.2);*/
+  border: none;
+  border-radius: 50%;
+  padding: 0;
+}
+
+.video__column {
+  width: 22.7%;
+}
+
+.stories {
+}
+
+.stories__title {
+  margin: 0;
+}
+
+.stories__menu {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.stories__column {
+  text-align: justify;
+  width: 26%;
+}
+
+.stories__foto {
+  width: 100%;
+}
+
+.stories__data {
+  text-align: right;
+  font-weight: bold;
+  opacity: 0.3;
+}
+
+.hashtag__foto {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.hashtag__img {
+  width: 14,3%;
+}
+
+.hashtag__img:nth-of-type(1n+2) {
+  margin-left: 0;
+  padding: 0;
+}
+
+.info {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0;
+  padding: 0;
+  background: #000;
+  color: #FFF;
+}
+
+img { max-width: 100%; }
```

Comment:
Codacy found an issue: [Expected selector "img" to come before selector "a img:hover" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Project.Axion/pullRequest?prid=5456827)